tests/shell/shell_recorder/tutorial_wrapper/CMakeLists.txt

```
@@ -32,6 +32,7 @@ add_msr_test (
 	ENVIRONMENT KDB_EXEC_PATH=${CMAKE_SOURCE_DIR}/scripts/kdb)
 add_msr_test (tutorial_storage_plugins "${CMAKE_SOURCE_DIR}/doc/tutorials/storage-plugins.md" REQUIRED_PLUGINS type yamlcpp)
 add_msr_test (cmerge "${CMAKE_SOURCE_DIR}/doc/tutorials/cmerge.md" REQUIRED_PLUGINS hosts line PROPERTY LABELS memleak)
+add_msr_test (tutorial_spec "${CMAKE_SOURCE_DIR}/doc/tutorials/specification.md")
```

```suggestion
add_msr_test (tutorial_spec "${CMAKE_SOURCE_DIR}/doc/tutorials/specification.md" PROPERTY LABELS memleak)
```

Seems like adding the `NO_SANITIZE` attributes didn't help. Please exclude the test from ASAN.